packages/amplify-frontend-javascript/lib/framework-config-mapping.js

```diff
@@ -49,12 +53,41 @@ const defaultConfig = {
   StartCommand: `${npm} run-script start`,
 };
 
-module.exports = {
-  angular: angularConfig,
-  ember: emberConfig,
-  ionic: ionicConfig,
-  react: reactConfig,
-  'react-native': reactNativeConfig,
-  vue: vueConfig,
-  none: defaultConfig,
-};
+function getAngularConfig(projectRoot) {
+  const angularConfigFile = path.join(projectRoot, 'angular.json');
+  const angularProjectConfig = JSONUtilities.readJson(angularConfigFile, { throwIfNotExist: false });
+  let dist = 'dist';
+  if (angularProjectConfig && angularProjectConfig.projects && Object.keys(angularProjectConfig.projects).length) {
+    const { defaultProject } = angularProjectConfig;
+    const projectConfig = angularProjectConfig.projects[defaultProject];
+    dist =
+      (projectConfig &&
+        projectConfig.architect &&
+        projectConfig.architect.build &&
+        projectConfig.architect.build.options &&
+        projectConfig.architect.build.options.outputPath) ||
+      'dist';
+  }
+  return {
+    ...angularConfig,
```

Comment:
is this supposed to be `angularProjectConfig`?

Reply:
No, `angularProjectConfig` is  project specific config from `angular.json` file. We are using this file to detect the distribution directory and nothing else